snakefmt/parser/syntax.py

```diff
@@ -125,31 +132,46 @@ def check_empty(self):
                 f"{self.line_nb}{self.keyword_name} has no keywords attached to it."
             )
 
+    @property
+    def effective_indent(self):
+        return max(0, self.cur_indent - self.target_indent)
+
     def get_next_queriable(self, snakefile):
         buffer = ""
-        newline = False
+        newline, used_name = False, True
```

Comment:
Please don't define multiple variables in a single line like this.

Reply:
Also `is_newline` might be a better variable name?

Reply:
why? if they are of simple type like this, seems fine?

Reply:
> Also `is_newline` might be a better variable name?

not quite, because it can be True without the token being a newline